src/client/ui/selector-inspector-panel/utils/get-rect-in-absolute-coordinates.js

```diff
@@ -0,0 +1,14 @@
+export function getRectInAbsoluteCoordinates (element) {
+    const rect       = element.getBoundingClientRect();
```

Comment:
I guess the such function already exists in `testcafe/core` or in `testcafe/utils` modules. Try to find and reuse it.

Reply:
I searched for a similar function but didn't find it.